.yamllint

```diff
@@ -0,0 +1,5 @@
+extends: relaxed
+
+rules:
+    line-length:
+        max: 250
```

Comment:
there's only one line that's > 200; we could probably fold it with `>` and set this a little tighter 🤷

Reply:
Is that the description for the `working-directory`? I think we can split that and drop this whole file. I'd prefer that.

Reply:
Got it. Yeah, I wasn't sure how strict we wanted to be on line length since sometimes with GHA, it's unavoidable. there are quite a lot of lines in the 150-200 range still.. I can make a PR that uses `yamlfix` to remove extra quotes, and fold the descriptions, but maybe it's better to do some of that separately, and then see where our max line length ends up?

It's `github/workflows/beaker.yml:250` - currently 201 characters
```
run: bundle exec metadata2gha --domain ${{ inputs.domain }} --pidfile-workaround ${{ inputs.pidfile_workaround }} --beaker-facter "${{ inputs.beaker_facter }}" --beaker-hosts "${{ inputs.beaker_hosts }}"
```
I think in this case, folding with > and putting it across multiple lines would work Ok probably

Reply:
@ekohl want to see what you think of it as it is now?